sdk/metricsadvisor/azure-ai-metricsadvisor/setup.py

```diff
@@ -66,6 +66,5 @@
     install_requires=[
         "azure-core<2.0.0,>=1.23.0",
         "msrest>=0.6.21",
-        'six>=1.11.0',
```

Comment:
We had a dependency on `six`, but didn't actually use it anywhere in the library? hmm 🤦‍♀️